interface/main/finder/dynamic_finder.php

```diff
@@ -76,6 +76,10 @@
         color: var(--danger);
         transform: translateX(-50%);
     }
+    .card {
+        border: 0;
+        border-radius: 0;
+    }
```

Comment:
@stu01509 Instead of this use BS4's `border-0` and `rounded-0` classes instead
[https://getbootstrap.com/docs/4.5/utilities/borders/](https://getbootstrap.com/docs/4.5/utilities/borders/)

Reply:
I know we can use the bootstrap utils class, but this issue cause by datatables style.

When the datatables loading it will show the loading icon, but this icon has a `card` style, so it looks like it has a little bit border, I didn't have any choice, Do you have any suggestions?

![image](https://user-images.githubusercontent.com/22230889/85904349-13069080-b83b-11ea-8a45-3289394d50ec.png)

Reply:
Change line 64 where the `loading` variable is to a BS4 spinner
[https://getbootstrap.com/docs/4.5/components/spinners/](https://getbootstrap.com/docs/4.5/components/spinners/)

Reply:
I try to change to the bootstrap spinner, but it still effects by `card` class.

Reply:
Did you try making the spinner smaller?
[https://getbootstrap.com/docs/4.5/components/spinners/#size](https://getbootstrap.com/docs/4.5/components/spinners/#size)

Reply:
I try to use the smaller it looks better.

Reply:
Doesn't look like you removed this css yet...

Reply:
Yup, although we change to the smaller spinner, but it still has a little bit border

![image](https://user-images.githubusercontent.com/22230889/85911676-0bee7b00-b859-11ea-9bc4-0766415c8748.png)

Reply:
@stu01509 go to line 73 of the file and remove `top: -20px;` from the CSS (this is what caused it to have a weird border)

Reply:
You might also want to add a loading shade of some sort on that CSS line since it probably would look better that way (that is if it works that way but I dunno)

Reply:
Update the commit.

I guess it didn't work in your testing. It may cause by some commit merged, but I still show in my testing, just a little bit position different.

![image](https://user-images.githubusercontent.com/22230889/86583052-335dea00-bfb5-11ea-9037-f65e4926b2b8.png)